src/index.scss

```diff
@@ -0,0 +1,7 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
+main {
+  display: block;
+}
+$feature-flags: (
```

Comment:
The feature flags should be set before the the styles are imported, see https://www.carbondesignsystem.com/tutorial/react-step-1#add-ui-shell-sass